libs/angular-remote-components/src/lib/components/slot/slot.component.ts

```diff
@@ -40,6 +70,13 @@ export class SlotComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.components$ = this.slotService.getComponentsForSlot(this.name)
+    this.updateDataSub = combineLatest([this._assignedComponents$, this._inputs$, this._outputs$]).subscribe(
```

Comment:
unsubscribing should not be needed: https://onecx.github.io/docs/guides/current/angular/general/guidelines.html#_asynchronous_behavior_observables